live-examples/js-examples/number-issafeinteger.html

```diff
@@ -0,0 +1,13 @@
+<pre>
+<code id="static-js">function warn(x) {
+  	if (Number.isSafeInteger(x)) return "Precision safe.";
```

Comment:
Missing opening and closing curly braces.

Reply:
OK - yes. Let's stick to that.